packages/convex-helpers/server/retries.ts

```diff
@@ -0,0 +1,164 @@
+/**
+ * This file defines helper functions that can be used to retry a
+ * Convex action until it succeeds. An action should only be retried if it is
+ * safe to do so, i.e., if it's idempotent or doesn't have any unsafe side effects.
+ */
+import {
+  FunctionReference,
+  FunctionVisibility,
+  Scheduler,
+  FunctionArgs,
+  OptionalRestArgs,
+  getFunctionName,
+  makeFunctionReference,
+  DefaultFunctionArgs,
+  internalMutationGeneric,
+} from "convex/server";
+import { v, ObjectType } from "convex/values";
+
+const DEFAULT_WAIT_BACKOFF = 10;
```

Comment:
I mentioned this above but I think 10ms is probably a mistake and we should start with 100ms. Doesn't matter a huge amount but might save them hammering a third party service too much.

If you disagree then just leave it as-is.